README.md

```diff
@@ -1,295 +1,202 @@
-
-A common flag that nearly all programs and commands accept is a standalone `h`,
-for "**h**elp".
+**Unix systems are _case sensitive_**! This is very important. Other systems
+such as Windows are case insensitive when it comes to referring to files. Unix
+systems is not like this. As such it is possible to have two or more files and
+directories with the same name but letters of different case.
 
 ```bash
-$ ls -h
+hello_world.rb
+HELLO_WORLD.rb
+hello_world.RB
 ```
 
-Single-character options can typically be combined with each other. For example,
-in the `ls` command, `h` is a suffix on the `l` flag meaning "**h**uman readable
-formats." They can be combined with `a` meaning "**a**ll information including.
-Try these three together:
-
-We can use the `ls -al` flag with ls so that we can see all of the files including
-hidden files in "long form".
-
-```bash
-$ ls -lah
-```
-And also:
+In the code snippet above, these are all distinct and separate files.
 
-```bash
-$ ls -l -a -h
-```
-Both are valid input options.
+### Making New Directories Using `mkdir`
 
-When you've entered `$ ls -lah` above, you should have a received a list of files
-including some that you hadn't seen from entering just `$ ls` before:
+Make directories with the command `mkdir`. We can make directories with the
+`mkdir` command:
 
 ```bash
-drwxr-xr-x   6 kellyegreene  staff   204B Jun  2 11:21 .
-drwxr-xr-x   5 kellyegreene  staff   170B May 28 15:52 ..
--rw-r--r--@  1 kellyegreene  staff   6.0K May 28 15:52 .DS_Store
-drwxr-xr-x  13 kellyegreene  staff   442B Jun  2 11:02 .git
--rw-r--r--   1 kellyegreene  staff    66B May 28 15:49 .learn
--rw-r--r--   1 kellyegreene  staff    11K Jun  2 11:21 README.md
+$ mkdir name_of_directory
 ```
 
-Notice that at the top of the file output there are a bunch of files that start with
-a `.`, like `.DS_Store`
+Now if you enter `ls` you should see the empty directory you just created in the
+working directory.
 
-Files like `.DS_Store` are not listed. That's because files that start with a `.`
-are _hidden_ files. Your `.bash_profile` is a hidden file in your home directory.
-If you want to see the hidden files you can add the `a` flag to `ls` by typing `$ ls -a`.
-
-**Note:** *Combining flags is only valid for single-letter options. A "long option"
-such as* `--force` *is defined with more than one character and must be entered with
-its own flag.*
+We can make a folder _and_ its parents if they do not exist already by using the
+`-p` option. The command `mkdir -p first_directory/next_directory/my_directory`
+would also create the first and next folder.
 
 ### Use `mv` to Move or Rename Files and Directories
 
-Move, or `mv` is a command that moves one or more files or directories from one place
-to another.  To move a file from the current directory to another location, enter a
-path as the third word on the command line.
+Move, or `mv` is a command that moves one or more files or directories from one
```

Comment:
What's wrong with the ordering of topics here? Are we referring to something that hasn't been taught yet?

Reply:
@sgharms I moved them around because there were places that segued into the next paragraph, but I changed a lot of text at the same time. Should we reorder?